core/src/main/scala/cats/syntax/reducible.scala

```diff
@@ -9,3 +9,24 @@ trait ReducibleSyntax extends Reducible.ToReducibleOps {
 final class NestedReducibleOps[F[_], G[_], A](private val fga: F[G[A]]) extends AnyVal {
   def reduceK(implicit F: Reducible[F], G: SemigroupK[G]): G[A] = F.reduceK(fga)
 }
+
+trait ReducibleSyntaxBinCompat0 {
+  implicit final def catsSyntaxReducibleOps0[F[_], A](fa: F[A]): ReducibleOps0[F, A] =
+    new ReducibleOps0[F, A](fa)
+}
+
+final class ReducibleOps0[F[_], A](val fa: F[A]) extends AnyVal {
+
+  /**
+   * Apply `f` to each element of `fa` and combine them using the
+   * given `SemigroupK[G]`.
+   *
```

Comment:
I think in order for this to be valid you need to open it with `{{` and close after, that's why it's not actually running the doctests.

Reply:
True, plus there must be a space after each `scala>`. Fixed now :)